getting-started.md

```diff
@@ -24,6 +24,12 @@ You should be able to install KinD and Knative bits with the following:
 ./hack/setup-kind.sh
 ```
 
+For a Mac,  please use this script (also please note that the airplay reciever uses the 5000 port and may need to be disabled details [here](https://developer.apple.com/forums/thread/682332)):
```

Comment:
nit s/reciever/receiver/

Also, maybe say that if you want to you can change the port the registry listens if you don't want to disable the airplay receiver.

For this though, you have to download the release and manually change it, or run sed through it?